specification/billing/resource-manager/Microsoft.Billing/preview/2019-10-01-preview/billing.json

```diff
@@ -2629,6 +2629,94 @@
           }
         }
       }
+    },
+	"/providers/Microsoft.Billing/billingAccounts/{billingAccountName}/customers/{customerName}/policies/default": {
+      "get": {
```

Comment:
>get [](start = 7, length = 3)

I noticed that there's no delete support here. Does the resource always exist? (As in GET always returns a successful response?)

Reply:
Yes, this resource always exist